ts-rs/src/export.rs

```diff
@@ -221,6 +222,14 @@ fn import_path(from: &Path, import: &Path) -> String {
     }
 }
 
+fn to_absolute_path(path: &Path) -> PathBuf {
+    let Ok(current_path) = std::env::current_dir() else {
+        return path.to_owned()
+    };
```

Comment:
I think we can safely unwrap this.

Reply:
I see... Nevermind my previous comment then xD

Reply:
I would still like to change diff_paths to return a `Result` and use `?` instead of `unwrap`ping though